tests/foreman/ui/test_registration.py

```diff
@@ -172,6 +177,8 @@ def test_positive_global_registration_end_to_end(
         cmd = session.host.get_register_command(
             {
                 'general.operating_system': default_os.title,
+                'general.organization': module_org.name,
+                'general.location': smart_proxy_location.name,
```

Comment:
Can we use `module_target_sat.ui_session()` here? also select these org and loc when session starts, so we can skip this values?

Reply:
Sure. Updated